nfpm.go

```diff
@@ -181,6 +181,7 @@ type RPM struct {
 	// https://www.cl.cam.ac.uk/~jw35/docs/rpm_config.html
 	ConfigNoReplaceFiles map[string]string `yaml:"config_noreplace_files,omitempty"`
 	Signature            RPMSignature      `yaml:"signature,omitempty"`
+	Ghosts               []string          `yaml:"ghosts,omitempty"`
```

Comment:
maybe GhostFiles instead (to go with ConfigNoReplaceFiles)?

```suggestion
	GhostFiles          []string          `yaml:"ghost_files,omitempty"`
```

Reply:
Sounds good, thanks for reviewing.  I'll make the changes.